server/src/main/java/io/unitycatalog/server/persist/SchemaRepository.java

```diff
@@ -219,7 +227,9 @@ public SchemaInfo updateSchema(String fullName, UpdateSchema updateSchema) {
           PropertyDAO.from(updateSchema.getProperties(), schemaInfoDAO.getId(), Constants.SCHEMA)
               .forEach(session::persist);
         }
+        schemaInfoDAO.setOwner(callerId);
```

Comment:
This doesn't seem correct -- `updateSchema` shouldn't be updating the owner to the caller.

Reply:
Ah.  Good catch.  Removing.